data/en/census_household.json

```diff
@@ -42,14 +42,8 @@
                     "title": "What is your address?",
                     "type": "General",
                     "answers": [{
-                            "id": "building",
-                            "label": "Building",
-                            "mandatory": false,
-                            "type": "TextField"
-                        },
-                        {
                             "id": "address-line-1",
-                            "label": "Address Line 1",
+                            "label": "Address Line 1 (Including your house number)",
```

Comment:
`house number` or should this be `house name or number`

Reply:
i have updated with your suggestion :)